src/Symfony/Bridge/ProxyManager/LazyProxy/Instantiator/LazyLoadingValueHolderFactoryV2.php

```diff
@@ -22,6 +23,11 @@ class LazyLoadingValueHolderFactoryV2 extends BaseFactory
 {
     private $generator;
 
+    public function getProxifiedClass(Definition $definition): ?string
+    {
+        return $this->getGenerator()->getProxifiedClass($definition);
```

Comment:
that looks weird to me. `getProxifiedClass(Definition $definition)` is not part of `ProxyGeneratorInterface` used as return type.

Reply:
I suggest keeping this logic in RuntimeInstantiator instead (or in a separate class if needed elsewhere)